### ISSUE TITLE
deleting rawAPI base component gives an empty value

### ISSUE THREAD

**Issue by Hoxolotl** [SimplyEdit/simplycode#66]
1. I added a raw API in a Base Component
2. called it "life, 
3. edited the code" then 
4. saved, then 
5. published, then
6. deleted the code INSIDE the raw API, leaving it empty <- just the code, I did NOT click "delete" in the UI
7. saved 
8. published
9. deleted the raw API in the component.
10. saved
11. published

Page script in index.html ( or generated.html) now breaks because the <script> has a missing value in a key-value pair:

```
var simplyRawApi = {
          "life" : 
        };
```

**Comment by ylebre** [comment]
Yes, that is what it does. On compile each method in the raw api ends up in that object, and each of them needs a value of some kind.
